timesketch/api/v1/resources/event.py

```diff
@@ -327,6 +327,201 @@ def get(self, sketch_id):
         return jsonify(schema)
 
 
+class EventAddAttributeResource(resources.ResourceMixin, Resource):
+    """Resource to add attributes to events."""
+
+    EVENT_FIELDS = ["_id", "_type", "_index", "attributes"]
+    ATTRIBUTE_FIELDS = ["attr_name", "attr_value"]
+    INVALID_ATTRIBUTES_NAMES = [
+        'datetime', 'timestamp', 'message', 'timestamp_desc']
+
+    MAX_EVENTS = 100000
+    MAX_ATTRIBUTES = 10
+    EVENT_CHUNK_SIZE = 1000
+
+    def _parse_request(self, flask_request):
+        """Validate and parse a POST request to add attributes.
+
+        Args:
+            flask_request (flask.Request): The unmodified request object.
+
+        Returns:
+            A dict with searchindex IDs as keys and the associated events as
+                values.
+        """
+        if not flask_request.is_json:
+            abort(
+                HTTP_STATUS_CODE_BAD_REQUEST,
+                "Request must be in JSON format."
+            )
+        form = flask_request.json
+        events = form.get("events")
```

Comment:
how about `events = flask_request.json.get...` directly since you are not using `form` anywhere else?

Reply:
Done